.github/workflows/ci.yml

```diff
@@ -11,23 +10,141 @@ on:
+        with:
+          name: m2-jsonschema2pojo-jdk${{ matrix.build-java }}
+          path: ~/.m2/repository/org/jsonschema2pojo
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: ${{ matrix.run.java }}
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v4
+        with:
+          gradle-version: ${{ matrix.run.gradle }}
+
+      - name: Build Gradle example (Android)
+        working-directory: jsonschema2pojo-gradle-plugin/example/android
+        run: gradle --warning-mode all --info assembleDebug
+
+  gradle-example-java:
+    needs: project-test
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        build-java: [8]
+        run:
+          - { java: 8,  gradle: '8.9' }
+    name: JDK ${{ matrix.build-java }} | Example Gradle (Java) | run on Java ${{ matrix.run.java }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Download m2 org.jsonschema2pojo (matching build-java)
+        uses: actions/download-artifact@v4
+        with:
+          name: m2-jsonschema2pojo-jdk${{ matrix.build-java }}
+          path: ~/.m2/repository/org/jsonschema2pojo
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: ${{ matrix.run.java }}
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v4
+        with:
+          gradle-version: ${{ matrix.run.gradle }}
+
+      - name: Build Gradle example (Java)
+        working-directory: jsonschema2pojo-gradle-plugin/example/java
+        run: gradle --warning-mode all --info build
+
+  maven-example:
+    needs: project-test
+    runs-on: ubuntu-latest
+    continue-on-error: true
+    strategy:
+      fail-fast: false
+      matrix:
+        build-java: [8]
+        run:
+          - { java: 8,  maven: '3.9.9' }
+    name: JDK ${{ matrix.build-java }} | Example Maven | run on Java ${{ matrix.run.java }}
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Download m2 org.jsonschema2pojo (matching build-java)
+        uses: actions/download-artifact@v4
+        with:
+          name: m2-jsonschema2pojo-jdk${{ matrix.build-java }}
+          path: ~/.m2/repository/org/jsonschema2pojo
+
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: ${{ matrix.run.java }}
+
+      - name: Setup Maven
+        uses: s4u/setup-maven-action@v1.18.0
+        with:
+          maven-version: ${{ matrix.run.maven }}
+
+      - name: Extract project version
+        id: get_version
+        run: echo "version=$(mvn help:evaluate -Dexpression=project.version -q -DforceStdout)" >> $GITHUB_OUTPUT
+
+      - name: Build Maven example
+        working-directory: jsonschema2pojo-maven-plugin/example
+        run: mvn -Djsonschema2pojo.version=${{ steps.get_version.outputs.version }}" package
```

Comment:
There is an unbalanced `"` on this line.  That is why the tests are stalling out.